pkg/gce-pd-csi-driver/node.go

```diff
@@ -505,16 +505,16 @@ func (ns *GCENodeServer) NodeExpandVolume(ctx context.Context, req *csi.NodeExpa
 }
 
 func (ns *GCENodeServer) GetVolumeLimits() (int64, error) {
-	var volumeLimits int64
-
 	// Machine-type format: n1-type-CPUS or custom-CPUS-RAM or f1/g1-type
 	machineType := ns.MetadataService.GetMachineType()
-	if strings.HasPrefix(machineType, "n1-") || strings.HasPrefix(machineType, "custom-") {
-		volumeLimits = volumeLimitBig
-	} else {
-		volumeLimits = volumeLimitSmall
+
+	smallMachineTypes := []string{"f1-micro", "g1-small", "e2-micro", "e2-small", "e2-medium"}
```

Comment:
What about using a map of machine types to volume size?

Reply:
there are only two volume size limits and only a very small subset are "small" so that is probably not necessary, see here for an interesting comparison of lookup times between slices and maps: https://www.darkcoding.net/software/go-slice-search-vs-map-lookup/ (breakeven point around 5 items).

The thing I want to avoid is a map for small things with all the value being `smallValue` and then "everything not in the map is big". Or else I would have to be exhaustive with the map which is error prone and repetitive